### PR TITLE
OSDOCS-9379#adding IAM permissions

### PR DESCRIPTION
**Pull request by brendan-daly-red-hat**
Versions:
4.15+

Issue:
https://issues.redhat.com/browse/OSDOCS-9379

Link to docs preview:
[Required GCP permissions for installer-provisioned infrastructure](https://85580--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-account.html#minimum-required-permissions-ipi-gcp_installing-gcp-account)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by brendan-daly-red-hat** [comment]
@jianli-wei, PTAL at the draft update.

File: modules/minimum-required-permissions-ipi-gcp.adoc [hunk 1]
```diff
@@ -183,7 +183,8 @@ If your organization’s security policies require a more restrictive set of per
 .Required IAM permissions for installation
 [%collapsible]
 ====
-* `iam.roles.get`
```
Comment (jianli-wei) [inline modules/minimum-required-permissions-ipi-gcp.adoc:186]: @brendan-daly-red-hat I think the iam.roles.get permission is still needed.

**Review by jianli-wei — CHANGES_REQUESTED** [review]

File: modules/minimum-required-permissions-ipi-gcp.adoc (hunk 1 above)
Comment (brendan-daly-red-hat) [reply modules/minimum-required-permissions-ipi-gcp.adoc:186]: @jianli-wei, Thanks, I re-added the iam.roles.get permission.

**Review by xenolinux — COMMENTED** [review]
LGTM

**Review by xenolinux — COMMENTED** [review]
Merge review LGTM

**Pull request merged**